File: python/src/commands/registry.py
from .echo import Echo
from .ping import Ping
from .set import Set
from .get import Get
from .config import Config
from .exists import Exists
from .delete import Delete
from .increment import Increment
from .decrement import Decrement
from .lpush import LPush
from .lrange import LRange
from .rpush import RPush
from .save import Save
from .publish import Publish

class RegistryCommands:
  @classmethod
  def commands(cls) -> list:
    return [
      Config,
      Decrement,
      Delete,
      Echo,
      Exists,
      Get,
      Increment,
      LPush,
      LRange,
      Ping,
      Publish,
      RPush,
      Save,
      Set
    ]

File: ruby/lib/redis/commands/registry.rb
require_relative './config'
require_relative './decr'
require_relative './del'
require_relative './echo'
require_relative './exists'
require_relative './get'
require_relative './incr'
require_relative './lpush'
require_relative './lrange'
require_relative './ping'
require_relative './publish'
require_relative './rpush'
require_relative './save'
require_relative './set'

module Redis
  module Commands
    module Registry
      REGISTERED_COMMANDS = [
        Redis::Commands::Config,
        Redis::Commands::Decr,
        Redis::Commands::Del,
        Redis::Commands::Echo,
        Redis::Commands::Exists,
        Redis::Commands::Get,
        Redis::Commands::Incr,
        Redis::Commands::LPush,
        Redis::Commands::LRange,
        Redis::Commands::Ping,
        Redis::Commands::Publish,
        Redis::Commands::RPush,
        Redis::Commands::Save,
        Redis::Commands::Set
    ].freeze
    end
  end
end